docs/tutorials/music.rst

```diff
@@ -1,29 +1,26 @@
-Music
------
+Muzyka
+------
 
-MicroPython on the BBC micro:bit comes with a powerful music and sound module.
-It's very easy to generate bleeps and bloops from the device *if you attach a
-speaker*. Use crocodile clips to attach pin 0 and GND to the positive and
-negative inputs on the speaker - it doesn't matter which way round they are
-connected to the speaker.
+MicroPython na BBC micro:bit zawiera bogaty w możliwości moduł muzyka i dźwięk. Dzięki niemu można w prosty sposób zaprogramować BBC micro:bit do wygenerowania 
+dźwięków o wysokim lub niskim tonie. Oczywiście, aby coś usłyszeć trzeba *podłączyć głośniki* - za pomocą krokodylków podłącz pin 0 oraz GND do wejść głośnika. Nie ma znaczenia który zacisk do którego wejścia zostanie podłączony.
```

Comment:
"dźwięków o wysokim lub niskim tonie" — rozumiem problem z tłumaczeniem "bleeps and bloops", ale może "piśnięć i innych odgłosów" brzmiałoby tu lepiej?

Reply:
"-" → "--"